package.json

```
@@ -27,15 +27,15 @@
     "check-format": "prettier --check .",
     "check-spelling": "cspell --config=.cspell.json \"**/*.{md,mdx,ts,mts,cts,js,cjs,mjs,tsx,jsx}\" --no-progress --show-context --show-suggestions",
     "clean": "nx run-many -t clean --parallel=20",
-    "format": "prettier --ignore-path=$PROJECT_CWD/.prettierignore --config=$PROJECT_CWD/.prettierrc.json --write $INIT_CWD",
+    "format": "prettier --ignore-path=$PROJECT_CWD/.prettierignore --config=$PROJECT_CWD/.prettierrc.json --write .",
```

They are not documented very well but **`pnpm`** uses **`$INIT_CWD`** same as `yarn` but instead of **`$PROJECT_CWD`** it uses [**`$PNPM_SCRIPT_SRC_DIR`**](https://github.com/pnpm/pnpm/blob/95b088c7a7356d60ee05a68d1e07dcc2029494e3/exec/lifecycle/src/runLifecycleHook.ts#L125-L126).

So now We can have something like this:

```suggestion
    "format": "prettier --ignore-path=${PNPM_SCRIPT_SRC_DIR}/.prettierignore --config=${PNPM_SCRIPT_SRC_DIR}/.prettierrc.json --write ${INIT_CWD}",
```